docs/UsersGuide.asciidoc

```diff
@@ -957,20 +957,26 @@ openQA frequently checks whether assets need to be removed according to
 the configured limits.
 
 To find out whether an asset should be removed, openQA determines by which
-job groups the asset is used. If at least one job within a certain job group
-is using an asset, the asset is considered to be used by that group.
+groups the asset is used. If at least one job within a certain job group is
```

Comment:
a *parent job group* is still a *job group* so I would keep this notion. A *group* as word standing on its own is a bit ambiguous. And one can use a specification *child job group* if a *job group* is within a *parent job group*. That would make a *job group* without *parent job group* an *orphan job group* – or *parentless* as you called them further below. But I think the word is simply orphan ;). Did we have this discussion already some months/years ago? ;)

Reply:
> a _parent job group_ is still a _job group_ 

No, it's not. It's a group of job groups not a group of jobs. Perhaps we better use 'job group parent' instead to make it clear.